service/src/main/kotlin/no/nav/su/se/bakover/service/utbetaling/UtbetalingServiceImpl.kt

```diff
@@ -123,9 +127,13 @@ internal class UtbetalingServiceImpl(
             sakId = sak.id,
             saksnummer = sak.saksnummer,
             fnr = sak.fnr,
-            utbetalinger = sak.utbetalinger,
+            eksisterendeUtbetalinger = sak.utbetalinger,
             behandler = request.saksbehandler,
-            opphørsDato = request.opphørsdato,
+            // TODO send med periode
```

Comment:
?

Reply:
Vurderte å ta det ut mot frontend i denne PRen også, men delte det i to..